doc/introduction.rst

```diff
@@ -0,0 +1,50 @@
+.. _Introduction to XBlocks:
+
+#############################
+Introduction to XBlocks
+#############################
+
+You use the XBlocks APIs to build components that course teams can use to
+create independent course components that work seamlessly with other components
+in an online course.
```

Comment:
"You build components that course teams use to create components", this seems confusing.